cypress/commons/utils/apiUtils.js

```diff
@@ -149,6 +149,28 @@ const interceptLaunchScenario = (stubbingOptions) => {
   return alias;
 };
 
+const interceptStopScenarioRun = () => {
+  const alias = forgeAlias('reqStopScenarioRun');
+  cy.intercept({ method: 'POST', url: API_REGEX.STOP_SCENARIO_RUN, times: 1 }, (req) => {
+    if (stub.isEnabledFor('LAUNCH_SCENARIO')) {
+      const scenarioRun = req.url.match(API_REGEX.SCENARIO_RUN);
+      const scenarioRunId = scenarioRun[1];
+      const scenarioId = stub.getScenarioRunById(scenarioRunId);
+
+      const scenarioLastRun = {
+        id: scenarioRunId,
+        endTime: null,
+        message: null,
+        estimatedDuration: null,
+      };
+      stub.patchScenario(scenarioId, { state: 'Failed', lastRun: scenarioLastRun });
```

Comment:
Something's odd here, because `stub.getScenarioRunById(scenarioRunId);`  will return the object of a scenario run, and not a scenario id. I don't think the line `stub.patchScenario(scenarioId [...]` will work.

Reply:
In fact, this part of code is useless. I will remove it.